src/js/components/Nav/Nav.tsx

```diff
@@ -198,39 +203,47 @@ const Nav: NavType & { Item: ItemType } = ({
     className,
     mods
   );
-
+  
   const navHeaderIconClass = getClassName("Nav-headerIcon");
 
   return (
-    <nav className={cname} style={style} {...otherProps}>
-      {headerItem ? (
-        <div
-          className="Nav-header u-textSemiBold"
+    <>
+      { (isFlyoutActive && includeOverlay) && 
+        <div className="Nav-overlay"
           onClick={() => setIsFlyoutActive(!isFlyoutActive)}
-        >
-          <div className={navHeaderIconClass}>
-            <Avatar src={headerItem.image} size="fill" />
-          </div>
-          <div className="u-sizeFill Nav-itemTitle u-spaceLeftSm">
-            <span className="Nav-itemTitle">{headerItem.title}</span>
-          </div>
-          {flyoutSections && (
-            <div className="u-size1of8 Nav-itemTitle u-textRight">
-              <Icon mods="u-fontSizeMd" name="down" />
+        ></div>
```

Comment:
Another minor nitpick (sorry): Since this element will have no children, can we use a self-closing tag instead?